bokeh/model.py

```diff
@@ -255,7 +255,21 @@ def _to_json_like(self, include_defaults):
                 that haven't been changed from the default.
 
         """
-        attrs = self.properties_with_values(include_defaults=include_defaults)
+        all_attrs = self.properties_with_values(include_defaults=include_defaults)
+
+        # If __subtype__ is defined, then this model may introduce properties
+        # that don't exist on __view_model__ in bokehjs. Don't serialize such
+        # properties.
+        subtype = getattr(self.__class__, "__subtype__", None)
+        if subtype is not None and subtype != self.__class__.__view_model__:
+            attrs = {}
+            for attr, value in all_attrs.items():
+                if attr in self.__class__.__dict__:
+                    continue
+                else:
+                    attrs[attr] = value
+        else:
+            attrs = all_attrs
```

Comment:
@bryevdv, this change is a big one. Pretty much all charts examples failed in this PR. We were serializing all properties, even if there was nothing in bokehjs. I'm still curious why defaults tests worked. Also, this may not be the best location for this code, but looking on how much logic is there in `properties*()` methods, I preferred not to touch them in this PR.